tstate/tstate_test.go

```diff
@@ -613,3 +613,328 @@ func TestCreateView(t *testing.T) {
 		require.ErrorIs(err, database.ErrNotFound, "value not removed from db")
 	}
 }
+
+func TestGetValuePermissions(t *testing.T) {
```

Comment:
nit: Feels like these tests that just check `state.Permission` should be in `state/keys_test`?